src/site/stages/build/data/vagovRedirects.json

```diff
@@ -128,5 +128,10 @@
     "domain": "www.va.gov",
     "src": "/HEALTHBENEFITS/apply/",
     "dest": "/health-care/how-to-apply/"
+  },
+  {
+    "domain": "www.va.gov",
+    "src": "/decision-reviews/get-help",
+    "dest": "/decision-reviews/get-help-with-review-request/"
```

Comment:
There's an HTML redirect for this page at vagov-content butit has a 508 issue. This line moves the redirect to be generated in Metalsmith instead, which gets rid of the 508 issue.